domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaygroup/DisplayGroupDisplayContextParamValidator.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.ccd.definition.store.domain.validation.displaygroup;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationError;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationResult;
+import uk.gov.hmcts.ccd.definition.store.repository.FieldTypeUtils;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.DisplayGroupCaseFieldEntity;
+
+@Component
+public class DisplayGroupDisplayContextParamValidator implements DisplayGroupCaseFieldValidator {
+
+    public static final String LIST_PREFIX = "#LIST(";
+    public static final String TABLE_PREFIX = "#TABLE(";
+
+    @Override
+    public ValidationResult validate(DisplayGroupCaseFieldEntity entity) {
```

Comment:
Just a question but shouldn't we also check if all nested fields in collection are simple type field? Wasn't that agreed with @amansinghb in the end?

Reply:
Yeah, but that and a few things will come as part of separate ticket, this piece was missing from the original ticket. hence added this.

Reply:
Thanks for clarification 👍